{"links": {"self": {"href": "data/repositories/osrf/subt/issues/224/comments/54616547.json"}, "html": {"href": "#!/osrf/subt/issues/224#comment-54616547"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/224.json"}}, "type": "issue", "id": 224, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "/subt/pose_from_artifact_origin after return from tunnel?"}, "content": {"raw": "Martin, I just conducted a quick experiment on the `Urban Qualification` world and I was not able to duplicate your issue.  I repeatedly called the `/subt/pose_from_artifact_origin` with the correct robot name while in the starting area successfully, left the starting area which caused the call to repeatedly fail, and then reentered the starting area which resulted in the call being successful again.", "markup": "markdown", "html": "<p>Martin, I just conducted a quick experiment on the <code>Urban Qualification</code> world and I was not able to duplicate your issue.  I repeatedly called the <code>/subt/pose_from_artifact_origin</code> with the correct robot name while in the starting area successfully, left the starting area which caused the call to repeatedly fail, and then reentered the starting area which resulted in the call being successful again.</p>", "type": "rendered"}, "created_on": "2019-10-31T13:52:08.738088+00:00", "user": {"display_name": "Arthur Schang", "uuid": "{1403972a-9b64-442b-a69c-d7e9557e1d87}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D"}, "html": {"href": "https://bitbucket.org/%7B1403972a-9b64-442b-a69c-d7e9557e1d87%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/initials/AS-0.png"}}, "nickname": "Arthur Schang", "type": "user", "account_id": "557058:14b7fd50-d65a-47a8-825c-fa89681d07bd"}, "updated_on": null, "type": "issue_comment", "id": 54616547}